.travis.yml

```diff
@@ -16,8 +19,10 @@ install:
   - if [[ $GROUP == 'js' ]]; then pip install 'tornado<4'; fi
   - cd ipython && git submodule update && pip install -e ".[notebook]" && cd ..
   - pip install -r requirements.txt
-  - pip install nose requests!=2.4.2
+  - pip install nose requests!=2.4.2 fabric
   - python setup.py install
   - if [[ $GROUP == 'js' ]]; then cp nbextensions/* $(ipython locate)/nbextensions/; fi
 script:
   - ./scripts/runtests $GROUP
+after_success:
+  - if [[ $GROUP != 'js' ]]; then fab travis_docs; fi
```

Comment:
Do you want to build only on non-PR and master branch like [here](https://github.com/ipython/ipython-website/blob/master/.travis.yml#L20) ?